doc/multi_group/multi_group_tutorial.rst

```diff
@@ -0,0 +1,111 @@
+Run multiple robot arms
+===============================
+.. image:: multi_arm_tutorial.gif
+   :width: 500px
+
+
+For running multiple robot arms in MoveIt, the following points need to be kept in mind :
+ 1. All robot arms should be defined in the same URDF.
+ 2. Each arm should have its own planning_group.
+ 3. If arms need to move in sync, they can be defined in a combined planning_group.
+
+But before starting with multiple arms, the concept of planning groups, there usage and significance of multiple planning groups for multiple end effectors
+needs to be understood clearly. kindly go through `this <https://answers.ros.org/question/321843/how-is-moveit-planning-group-supposed-to-be-used/>`_ answer if you are a beginner/new to these concepts.
```

Comment:
As mentioned in the comment, please use a spell checker to fix grammar etc. throughout the text